python/lsst/pipe/tasks/photoCal.py

```diff
@@ -417,6 +417,9 @@ def run(self, exposure, sourceCat, expId=0):
         flux0 = 10**(0.4*r.zp)  # Flux of mag=0 star
         flux0err = 0.4*math.log(10)*flux0*r.sigma  # Error in flux0
         photoCalib = makePhotoCalibFromCalibZeroPoint(flux0, flux0err)
+        self.log.info("Photometric calibration factor (nJy/ADU): %f +/- %f",
+                      photoCalib.getCalibrationMean(),
+                      photoCalib.getCalibrationErr())
```

Comment:
or use `self.log.verbose` if this is not really a debug message but also not something you want people to see all the time. Pipeline batch execution runs at VERBOSE level rather than INFO.

Reply:
PhotoCalTask is very much not chatty:

* Are color terms being applied (and if so, from where)?
* Report result as a magnitude zero point.
* Report result as calibration factor (this addition).

We still don't have clear guidance on what to output at what level, but two lines summarizing what the result is seems appropriate here.

Reply:
I don't have a very strong opinion.  I just think people tend to look at task-level log messages in order to:
 - help diagnose a problem
 - get a _quick_ sense of status or timing (there are better ways for both, but probably not easier ways)

and I don't see this one as helping with either.